src/Libraries/PythonNodeModelsWpf/PythonNode.cs

```diff
@@ -39,6 +40,10 @@ public void CustomizeView(PythonNode nodeModel, NodeView nodeView)
             nodeView.MouseDown += view_MouseDown;
             nodeModel.DeletionStarted += NodeModel_DeletionStarted;
             nodeModel.Disposed += NodeModel_Disposed;
+
+            nodeView.PresentationGrid.Visibility = Visibility.Visible;
+            nodeView.PresentationGrid.DataContext = this.pythonNodeModel;
+            nodeView.PresentationGrid.Children.Add(new EngineLabel());
```

Comment:
Cool. You may want to merge the latest debug modes change in and put this block under debug mode

Reply:
@QilongTang Do we only want the `Engine Label` to be displayed in debug mode?

Reply:
@SHKnudsen The original scope is that the labelas well as combo box on editor, we make them display only under the debug mode. After some thoughts, I think the lable maybe fine but can we at least put the switcher under debug mode, you may need to add a converter for visibility of it. Other than that, this PR looks good.